### PR TITLE
Include connection template field values in components:dev:run output

### PR DESCRIPTION
**Pull request by taylorreece**
The `prism components:dev:run` command pulls down a test instance connection's values, so you can use them locally. This is handy if you have a basic auth, API key, or OAuth 2.0 connection in a test instance, and you want to use that connection's API key, etc., to test actions locally or to make requests to the third-party app from your machine.

Currently, if you run `prism components:dev:run` and the connection you're fetching leverages a [connection template](https://prismatic.io/docs/connections/#connection-templates), the connection template's fields are omitted from the values you receive back.

image

This changes the query the command makes, so that connections' templated values are fetched. Then, templated connection values are merged with other input field values, so the user receives all connection-relevant information.

image

### CODE REVIEW

File: src/commands/components/dev/run.ts [hunk 1]
```diff
@@ -87,15 +102,21 @@ export default class RunCommand extends Command {
...
 
-    const { meta, inputs } = connection;
+    const { meta, inputs, requiredConfigVariable } = connection;
 
-    const fields = inputs.nodes.reduce<Record<string, unknown>>(
-      (result, { name, value }) => ({ ...result, [name]: value }),
-      {},
-    );
+    // Combine templated connection field values with the test instance's field values
+    const fields = {
```
Comment (pattra) [inline src/commands/components/dev/run.ts:111]: Clarification question here -- this may be an edge case or not applicable at all, so just LMK if that's the case.

Is it possible for `connectionTemplate` inputs and regular inputs to share keys/names? And if so, do we care about how merge conflicts are resolved?

The reason I ask is b/c if a `connectionTemplate` input and regular input share a key here (`someSharedKey`), this merge results in the regular inputs winning out. If for whatever reason `inputs.nodes.someSharedKey` comes back with an `undefined` value in the query response (as opposed to being omitted), that results in `someSharedKey` being undefined in the result, regardless of what's in the template.

Comment (taylorreece) [reply src/commands/components/dev/run.ts:111]: That's a great question. With the way the API is currently implemented, an input value will either be contained within the connection template, or within the instance's values, but not both. So, I don't believe we'll have merge conflicts.

**Review by pattra — APPROVED** [review]
